lssinglestore/singlestore.go

```diff
@@ -185,7 +185,7 @@ func (p *SingleStore) CreateSchemaTableIfNotExists(ctx context.Context, _ *inter
 	}
 	if schema != "" {
 		_, err := d.DB().ExecContext(ctx, fmt.Sprintf(`
-				CREATE DATABASE IF NOT EXISTS %s
+				CREATE DATABASE IF NOT EXISTS %s PARTITIONS 2
```

Comment:
Does it make sense that the database where store the set of migrations that have run should have 2 partitions in production?

It's only accessed during program startup and is very tiny.

Reply:
Yeah, if it's really small, 2 partitions is the way to go (it does mean that database shouldn't be used for anything else), since that's the minimum singlestore supports today.